operator/apis/machinelearning/v1/seldondeployment_types.go

```diff
@@ -185,8 +196,9 @@ type PredictorSpec struct {
 	Traffic         int32                   `json:"traffic,omitempty" protobuf:"bytes,9,opt,name=traffic"`
 	Explainer       *Explainer              `json:"explainer,omitempty" protobuf:"bytes,10,opt,name=explainer"`
 	Shadow          bool                    `json:"shadow,omitempty" protobuf:"bytes,11,opt,name=shadow"`
-	Protocol        Protocol                `json:"protocol,omitempty" protobuf:"bytes,12,opt,name=protocol"`
-	Transport       Transport               `json:"transport,omitempty" protobuf:"bytes,13,opt,name=transport"`
+	Default         bool                    `json:"default,omitempty" protobuf:"bytes,12,opt,name=default"`
```

Comment:
Do we need to change the CRD to add default? As it really not anything that changes functionality

Reply:
I thought that might be a contentious addition. Happy to remove it if there is no use for it. Could I add logic to say that if a service is not a shadow it automatically gets labelled as a default instead in that case?

Reply:
Have replaced the Default with Canary, which is a more useful flag

Reply:
Removed new field entirely now and replaced with logic based on Traffic %